### PR TITLE
NA speakers announcement

### PR DESCRIPTION
**Pull request by MissMikeyMay**
Draft speaker announcement, also converted the previous announcement from md to rst and fixed up the news items listings.

### CODE REVIEW

File: docs/conf/na/2017/news/announcing-presentations.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+If you haven't purchased your ticket yet, time hasn't quite run out. Check out the `Tickets <http://www.writethedocs.org/conf/na/2017/tickets/>`_ page for more details.
+**More than half the tickets are already sold out**, and we expect to sell out before the end of April, so don't wait to get your tickets!
+
+Prague Conference Update
+------------------------
+
+In case your proposal wasn't selected for our Portland conference, or if you (like some of our friends) were waiting for this announcement, we are happy to also launch our `website and ticket sales http://www.writethedocs.org/conf/eu/2017/news/announcing-website-tickets/`_ for the conference that will be held on **September 10-12** in Prague, Czech Republic.
+
+The call for proposals will open soon, sign up for our mailing list and stay tuned!
```
Comment (plaindocs) [inline docs/conf/na/2017/news/announcing-presentations.rst]: "stay tuned" ? :-p

**Comment by plaindocs** [comment]
Apologies for the extra noise. :-p Thought I'd merge into `2017-speakers-na` but that adds a TON of commits to things.

**Comment by MissMikeyMay** [comment]
@plaindocs what's wrong with stay tuned? We haven't published the CFP for EU yet ;)

**Comment by plaindocs** [comment]
Current list of titles.
```
"Writing the Docs, University Edition"
"FOSS it up, with women !"
"Everyone's a player (in a mid-90s MUD)"
"Testing: it's not just for code anymore"
"Write the Docs Meetup Groups"
"Building navigation for your doc site: 5 best practices"
"You have already succeeded: Design critique guidelines make feedback easier."
"Only Interesting Conversations: The symbiotic relationship between docs and support"
"QueerTea"
"Error Messages: Being Humble, Human, and Helpful will make users Happy"
"Interviewing and hiring technical writers: the Siberian way"
"Treating documentation like code: a practical account"
"No Community Members Were Harmed in the Making of This Doc Sprint: How we ran a 48-hour event to collect community wisdom into a guidebook for newsroom developers"
"The Wisdom of Crowds: Crowdsourcing Minimalism in an Open Organization "
"Caring Systems: Documentation as care"
"Bootstrapping Docs at a Startup"
"Release Notes: What's Working (and Not Working) for You?"
"Worst-case Scenario Planning"
"Even Naming This Talk Is Hard"
"Start with the tasks, not the endpoints"
"Intelligent Documents and the Verifiability Crisis in Science, Tech Writing, and Life"
"Do you know a runbook from a flip book? How sysadmins use documentation"
```

**Comment by thatdocslady** [comment]
Hahahaha I see I used my other GH user for this. Oops. Anyway.

**Pull request merged**